### PR TITLE
Add etcdbackup featuregate

### PR DESCRIPTION
**Pull request by hasbro17**
Splitting out the featuregate for the EtcdBackup API from https://github.com/openshift/api/pull/1482

/cc @deads2k @tjungblu

### CODE REVIEW

File: config/v1/feature_gates.go [hunk 1]
```diff
@@ -271,4 +271,13 @@ var (
 		ResponsiblePerson:   "rvanderp3",
 		OwningProduct:       ocpSpecific,
 	}
+	FeatureGateEtcdBackupAPI = FeatureGateName("EtcdBackupAPI")
```
Comment (deads2k) [inline config/v1/feature_gates.go:274]:
```suggestion
	FeatureGateEtcdBackupAPI = FeatureGateName("AutomatedEtcdBackup")
```

Comment (deads2k) [reply config/v1/feature_gates.go:274]: after that lgtm

**Comment by hasbro17** [comment]
@deads2k This should be good to merge if you can take another look.

**Pull request merged**